### PR TITLE
0.12.0rc0

### PR DESCRIPTION
**Pull request by SLdragon**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 # Changelog
 
 [![PyPI version](https://badge.fury.io/py/iotedgehubdev.svg)](https://badge.fury.io/py/iotedgehubdev)
+## 0.12.0 - 2019-11-xx
+### Added
+* Add cert extensions required by iot edge transparent gateway
```
Comment (adashen) [inline CHANGELOG.md]: It's not added. How about move to Changed just say "Update cert generation logic"?

Comment (SLdragon) [reply CHANGELOG.md]: OK

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,14 @@
...
+
+### Changed
+* Add python version requirement in README and setup.py
```
Comment (adashen) [inline CHANGELOG.md]: Specify here that we now only several versions of python?

Comment (SLdragon) [reply CHANGELOG.md]: OKOK

File: .vscode/launch.json [hunk 3]
```diff
@@ -12,9 +12,9 @@
             "module": "iotedgehubdev.cli",
             "args": [
                 // "singlemoduletest"
-                "setup",
-                "--connectionstring",
-                "<DeviceConnstr>"
+                "start",
```
Comment (adashen) [inline .vscode/launch.json]: need check in this file?

Comment (SLdragon) [reply .vscode/launch.json]: Oh, just reverted

File: .vscode/settings.json [hunk 4]
```diff
@@ -1,4 +1,8 @@
 {
-    "python.unitTest.unittestEnabled": true,
-    "python.unitTest.pyTestEnabled": true,
+    "python.testing.unittestEnabled": true,
+    "python.testing.pytestEnabled": true,
+    "files.associations": {
+        "*.spec": "python"
+    },
+    "python.pythonPath": "C:\\Python27\\python.exe"
```
Comment (adashen) [inline .vscode/settings.json]: shall we add this file?

**Pull request merged**